lib/schema.js

```diff
@@ -164,7 +164,7 @@ const linkSchemas = schemas => {
           ));
         }
       } else if (field.category) {
-        const category = categories.get(field.category);
+        const category = categories.get(field.category).definition;
```

Comment:
If there will be no such category in `categories` this will throw an error. Maybe it is better to do like this
```javascript
if (category) {
  const { definition } = category;
  ...
}
```